server/jetstream_cluster_test.go

```diff
@@ -10528,3 +10528,89 @@ func TestJetStreamClusterNewHealthz(t *testing.T) {
 	sl = c.restartServer(sl)
 	c.waitOnServerHealthz(sl)
 }
+
+func TestJetStreamClusterConsumerOverrides(t *testing.T) {
+	c := createJetStreamClusterExplicit(t, "JSC", 3)
+	defer c.shutdown()
+
+	nc, js := jsClientConnect(t, c.randomServer())
+	defer nc.Close()
+
+	_, err := js.AddStream(&nats.StreamConfig{
+		Name:     "TEST",
+		Subjects: []string{"foo"},
+		Replicas: 3,
+	})
+	require_NoError(t, err)
+
+	// Test replica override.
+	// Make sure we can not go "wider" than the parent stream.
+	ccReq := CreateConsumerRequest{
+		Stream: "TEST",
+		Config: ConsumerConfig{
+			Durable:   "d",
+			AckPolicy: AckExplicit,
+			Replicas:  5,
+		},
+	}
+	req, err := json.Marshal(ccReq)
+	require_NoError(t, err)
+
+	ci, err := nc.Request(fmt.Sprintf(JSApiDurableCreateT, "TEST", "d"), req, time.Second)
+	require_NoError(t, err)
+
+	var resp JSApiConsumerCreateResponse
+	err = json.Unmarshal(ci.Data, &resp)
+	require_NoError(t, err)
+
+	if resp.Error == nil || resp.Error.ErrCode != 10126 {
```

Comment:
Would do: 
```
if resp.Error == nil || !IsNatsErr(resp.Error, JSConsumerReplicasExceedsStream) {
``` 
instead of checking err code directly..

Reply:
ok done.